pkg/release/release.go

```diff
@@ -178,6 +180,10 @@ const (
 	TestAction          action = "test"
 )
 
+const (
+	Migrate2to3Annotation string = "helm.fluxcd.io/migrate-2to3"
```

Comment:
```suggestion
	MigrateAnnotation string = "helm.fluxcd.io/migrate"
```

By keeping the annotation generic we allow a future v3 -> v4 to be added without too much fuss.